docs/input/beckhoff-ads-community.md

```
@@ -6,65 +6,242 @@ This input only supports symbols and not direct addresses.
+    readType: 'notification'
+    symbols:
+      - "MAIN.MyVariable"
+```
+
+#### Configuration Parameters
+
+| Parameter | Required | Default | Description |
+|-----------|----------|---------|-------------|
+| **targetIP** | Yes | — | IP address of the Beckhoff PLC |
+| **targetAMS** | Yes | — | AMS net ID of the target |
+| **symbols** | Yes | — | List of symbols to read from (see [Symbol Format](#symbol-format) below) |
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)